website/src/pages/index.js

```diff
@@ -14,269 +17,144 @@ function getBanner() {
-            <div className="col col--4">
-              <div className="card large light">
-                <div className="card__header">
-                  <h3>Docs</h3>
-                </div>
-                <div className="card__body">
-                  <p>
-                    The core concepts of dbt, from models, to sources, to tests.
-                  </p>
-                </div>
-                <div className="card__footer">
-                  <Link
-                    className="button button--primary"
-                    to="/docs/introduction">
-                    Read Up
-                  </Link>
-                </div>
-              </div>
+            <div className="featured-header"><h2>Featured resource</h2></div>
+            <div className="featured-resource">
+            <BlogPostCard postMetaData={featuredResource} />
             </div>
-            <div className="col col--4">
-              <div className="card large light lighter">
-                <div className="card__header">
-                  <h3>Reference</h3>
-                </div>
-                <div className="card__body">
-                  <p>
-                    The technical reference for dbt configurations. You’ll need to know the basics of dbt before using this section.
-                  </p>
-                </div>
-                <div className="card__footer">
-                  <Link
-                    className="button button--primary"
-                    to={useBaseUrl('reference/dbt_project.yml')}>
-                    Browse Reference
-                  </Link>
-                </div>
-              </div>
-            </div>
-            <div className="col col--4">
-              <div className="card large light lightest">
-                <div className="card__header">
-                  <h3>FAQs</h3>
-                </div>
-                <div className="card__body">
-                  <p>
-                    Commonly asked questions about dbt.
-                  </p>
-                </div>
-                <div className="card__footer">
-                  <Link
-                    className="button button--primary"
-                    to={useBaseUrl('/docs/faqs')}>
-                    Get Answers
-                  </Link>
-                </div>
+          </section>
+
+          <section className="from-the-blog">
+            <h2>The latest from the Developer Blog</h2>
+            <PostCarousel blogPostData={recentBlogData} />
+          </section>
+
+          <section className="from-the-community">
+            <h2>From the dbt community</h2>
+            <div className="grid--3-col">
+              <div>
+                <Card
+                  title="Join the community"
+                  body="Connect with data practitioners from around the world."
+                  link="https://www.getdbt.com/community/join"
+                  icon="smiley-face"
```

Comment:
This card is going to a 404 page, I believe the link here would be `https://www.getdbt.com/community/join-the-community/`

Reply:
Oh, just noticed there's a new `community/join` page in a separate Docs PR: https://deploy-preview-2069--docs-getdbt-com.netlify.app/community/join. So it might point to this instead?

Reply:
Yep that is it! :D

Reply:
Awesome, then @jasnonaz's suggestion looks like it'll be correct for that card!